src/main/kotlin/org/rust/lang/core/resolve/ref/RsLitExprReferenceContributor.kt

```diff
@@ -190,3 +206,32 @@ private class RsLiteralWebReferenceProvider : PsiReferenceProvider() {
         private val SPACE_SPLITTER: Regex = Regex("\\s")
     }
 }
+
+/**
+ * Current support:
+ *
+ * [rust-lang](https://github.com/rust-lang/rust)
+ * - [RsInnerAttr] / [RsOuterAttr] - `unstable` - `issue`
+ * - [RsInnerAttr]? / [RsOuterAttr] - `rustc_const_unstable` - `issue`
+ */
+private class RsLiteralGitHubIssueProvider : RsLiteralWebReferenceProviderBase() {
+
+    override fun getReferencesByStringLiteral(expr: RsLitExpr, literal: RsLiteralKind.String): Array<PsiReference> {
+        if (expr.containingCrate.origin != PackageOrigin.STDLIB) return PsiReference.EMPTY_ARRAY
+
+        val exprMetaItem = expr.parent as? RsMetaItem ?: return PsiReference.EMPTY_ARRAY
+
+        if (exprMetaItem.name != "issue") return PsiReference.EMPTY_ARRAY
+
+        val issueNumber = literal.value?.toLongOrNull() ?: return PsiReference.EMPTY_ARRAY
+        val attr = exprMetaItem.parentOfType<RsAttr>() ?: return PsiReference.EMPTY_ARRAY
+
+        return if (attr.metaItem.name in RUST_ISSUE_ATTR_NAMES) {
```

Comment:
Actually, it doesn't work if you wrap attribute into `cfg_attr` because in this case `attr.metaItem.name` returns `cfg_attr`. I understand that it's quite a rare case and I found only one example in stdlib (with `none` as issue value but anyway) but let's support it as well.

I think you need to do 2 things:
- find parent meta item of `exprMetaItem` (something like `val parentMetaItem = (exprMetaItem.parent as? RsMetaItemArgs)?.parent as? RsMetaItem` and check its name
- ensure that this parent is top-level attribute. You can do it with `isRootMetaItem` extention method. It takes into account all possible parent `cfg_attr` attributes

And let's add the corresponding test